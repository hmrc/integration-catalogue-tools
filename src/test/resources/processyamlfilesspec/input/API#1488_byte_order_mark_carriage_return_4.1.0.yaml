openapi: 3.0.3
info:
  title: 'API#1488 has byte order mark and carriage return line feed'
  description: This API provides the capability.
  version: 4.1.0
  contact:
    name: Joe Bloggs
    email: joe.bloggs@example.com
servers:
  - url: 'https://hostname'
    description: Actual environment values can be obtained
tags:
  - name: 'API#1488'
    description: This API provides the capability.
