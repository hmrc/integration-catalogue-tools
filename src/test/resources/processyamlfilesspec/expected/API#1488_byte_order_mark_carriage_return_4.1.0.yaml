openapi: 3.0.3
info:
  title: 'API#1488 has byte order mark and carriage return line feed'
  description: This API provides the capability.
  version: 4.1.0
  contact:
    name: Joe Bloggs
    email: joe.bloggs@example.com
  x-integration-catalogue:
    reviewed-date: 2022-04-22T20:27:05Z
    platform: CORE_IF
    publisher-reference: 1488
servers:
  - url: 'https://hostname'
    description: Actual environment values can be obtained
tags:
  - name: 'API#1488'
    description: This API provides the capability.
